tests/unit/visual/tst_combobutton.11.qml

```diff
@@ -178,9 +178,7 @@ Item {
         }
 
         function test_flickRectCombo() {
-            // skip the test temporarily due to flakyness on powerpc target
             // https://bugs.launchpad.net/ubuntu-ui-toolkit/+bug/1315244
-            skip("The test is flaky on powerpc target");
```

Comment:
Yeah, link removed. Don't think we need to do a skip for powerpc here, since we don't build on it anyway. So let's leave that for when we get the rename to lomiri done and we resolve any differences between here and `lomiri-ui-toolkit` repo on gitalb, for when debian needs ppc builds.